test/react-native-cli/features/automate-symbolication.feature

```diff
@@ -0,0 +1,133 @@
+Feature: automate symbolication command
+
+Scenario: successfully modify project
+    When I run the React Native service interactively
+    And I input "bugsnag-react-native-cli automate-symbolication" interactively
+    Then I wait for the shell to output a line containing "No repo detected." to stdout
+    And I wait for the shell to output the following to stdout
+        """
+        This command may make modifications to your project. It is recommended that you commit the
+        current status of your code to a git repo before continuing.
+        """
+    And I wait for the current stdout line to contain "Do you want to continue anyway?"
+    When I input "y" interactively
+    And I wait for the current stdout line to contain "Do you want to automatically upload source maps as part of the Xcode build?"
+    When I press enter
+    And I wait for the current stdout line to contain "Do you want to automatically upload source maps as part of the gradle build?"
+    When I press enter
+    And I wait for the shell to output the following to stdout
+      """
+      Using yarn or npm?
+      """
```

Comment:
Tried rebasing this onto the integration branch and these now fail with the new Gradle plugin version check just before here